pom.xml

```diff
@@ -14,7 +14,8 @@
     <description>Allows Jenkins administrators to control what in-process scripts can be run by less-privileged users.</description>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Script+Security+Plugin</url> 
     <properties>
-      <jenkins.version>2.7.3</jenkins.version>
+      <jenkins.version>2.60.3</jenkins.version>
```

Comment:
Ugh, this is a problem for me. We'll have to bump everything downstream of this as well, which means...well, everything. I'm going to have to be persuaded to approve this one.

Reply:
Yeah, it's going to have to justify its worth by fixing hangs and offering *really* big performance boosts -- would still like to cut at least one more script security release before we go to mandatory Java 8.  

Worst case we can deal with the pain of backports if it is worth it.